core/src/test/kotlin/api/ProductsRouteIntegrationTest.kt

```diff
@@ -676,4 +695,231 @@ class ProductsRouteIntegrationTest : AbstractIntegrationTest({
             }
         }
     }
+
+    "PUT/DELETE /products/{productId}/groups/{groupId}" should {
+        forAll(
+            row(HttpMethod.Put),
+            row(HttpMethod.Delete)
+        ) { method ->
+            "require ProductPermission.WRITE for method '${method.value}'" {
+                val createdProd = createProduct()
+                val user = Username(TEST_USER.username.value)
+                print("user: $user")
+                print("createdOrg: $createdProd")
+                print("createdOrg.id: ${createdProd.id}")
```

Comment:
We should not do console output like this in tests. Instead, consider regular logging, or [withClue()](https://kotest.io/docs/assertions/clues.html) is only required in the context of a failed assertion.

Reply:
That was debug output I forgot to remove, I've removed it now 🙂